### PR TITLE
Feature/display users name

### PR DESCRIPTION
**Pull request by avivbarel**
Solves #39

### CODE REVIEW

File: src/utils/auth.js [hunk 1]
```diff
@@ -66,7 +65,7 @@ export async function validateLogin () {
     if (getCookie("kd-token")) {
       parseToken();
     } else {
-      window.open(`/auth/login`, '_self');
```
Comment (yonatandt) [inline src/utils/auth.js]: Don't commit those changes.
We need to find a solution for that.

File: src/utils/constants.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const name = 'KDrive';
 const disableExternal = false;
-const baseURL = '/';
```
Comment (yonatandt) [inline src/utils/constants.js]: Same

File: src/utils/constants.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ const config = {
 };
 
 export async function fetchConfig() {
-	const res = await fetch(`${baseURL}api/config`);
```
Comment (yonatandt) [inline src/utils/constants.js]: same

**Review by yonatandt — CHANGES_REQUESTED** [review]

File: src/components/Sidebar.vue [hunk 4]
```diff
@@ -1,9 +1,9 @@
 <template>
   <nav :class="{active}">
     <template v-if="isLogged">
-      <router-link class="action" to="/files" :aria-label="$t('sidebar.myFiles')" :title="$t('sidebar.myFiles')">
+      <router-link class="action" to="/files" :aria-label="$t('sidebar.myFiles')" :title="$t('sidebar.myFiles.title')">
         <i class="material-icons">folder</i>
-        <span>{{ $t('sidebar.myFiles') }}</span>
+        <span>{{ $t('sidebar.myFiles.personalized', {person: user.firstName}) }}</span>
```
Comment (omerzamir) [inline src/components/Sidebar.vue]: add condition on personalized to check if exist, if not fallback to title

**Review by omerzamir — CHANGES_REQUESTED** [review]

File: src/components/Header.vue [hunk 5]
```diff
@@ -183,6 +186,11 @@ export default {
     },
     resetPrompts () {
       this.$store.commit('closeHovers')
+    },
```
Comment (Shahar-Y) [inline src/components/Header.vue]: Delete the code duplication

File: src/components/Header.vue [hunk 6]
```diff
@@ -89,10 +91,11 @@ export default {
     UploadButton,
     SwitchButton,
     MoveButton,
-    ShellButton
+    ShellButton,
+    UserButton
   },
   data: function () {
-    return {
+    return {
```
Comment (Shahar-Y) [inline src/components/Header.vue:98]: delete space

File: src/utils/auth.js (hunk 1 above)
Comment (avivbarel) [reply src/utils/auth.js]: done

File: src/utils/constants.js (hunk 3 above)
Comment (avivbarel) [reply src/utils/constants.js]: done

File: src/utils/constants.js (hunk 2 above)
Comment (avivbarel) [reply src/utils/constants.js]: done

File: src/components/Sidebar.vue (hunk 4 above)
Comment (avivbarel) [reply src/components/Sidebar.vue]: done

**Review by Shahar-Y — APPROVED** [review]
Approved as far as I'm concerned

**Review by omerzamir — APPROVED** [review]

**Pull request merged**